tests/testthat/test-download.R

```diff
@@ -114,7 +114,7 @@ test_that("downloads work with file URIs", {
 
 test_that("downloads work with UNC paths on Windows", {
   skip_on_cran()
-  skip_if_not(renv_platform_windows())
+  skip_on_os(c("linux", "mac"))
```

Comment:
`skip_unless_os("windows")`?

Reply:
I would prefer the full set, but I understand if you don't want to -- `skip_unless_os()` would make me happy enough.

Reply:
I think it's easiest to start by adding the per OS helpers to renv, and I'll see how it feels.